### PR TITLE
fix docs tom file

### PR DESCRIPTION
**Pull request by jd-lara**
Thanks for opening a PR to PowerNetworkMatrices.jl, please take note of the following when making a PR:

Check the [contributor guidelines](https://nrel-sienna.github.io/PowerNetworkMatrices.jl/stable/code_base_developer_guide/developer/)

### CODE REVIEW

**Comment by jd-lara** [comment]
Preview https://nrel-sienna.github.io/PowerNetworkMatrices.jl/previews/PR52/

File: docs/src/tutorials/tutorial_Incidence_BA_ABA_matrices.md [hunk 1]
```diff
@@ -41,12 +41,12 @@ incidence_matrix.axes
...
 
-# lookup: dictionary linking the branche names and bus numbers with the row 
+# lookup: dictionary linking the branche names and bus numbers with the row
 # and column numbers, respectively.
 incidence_matrix.axes
 
 # ref_bus_positions: set containing the positions of the reference buses.
-# this represents the positions where to add the column of zeros. Please refer to the 
+# this represents the positions where to add the column of zeros. Please refer to the
 # exaple in the BA matrix for more details.
```
Comment (rodrigomha) [inline docs/src/tutorials/tutorial_Incidence_BA_ABA_matrices.md:50]: typo: example

File: docs/src/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ CurrentModule = PowerNetworkMatrices
 ## Overview
 
 `PowerNetworkMatrices.jl` is a [`Julia`](http://www.julialang.org) package for
-the evaluation of network matrices given the system's data. The package allows to compute 
+the evaluation of network matrices given the system's data. The package allows to compute
 the matrices according to different methods, providing a flexibe and powerful tool.
```
Comment (rodrigomha) [inline docs/src/index.md:11]: typo: flexible

File: docs/src/tutorials/tutorial_Incidence_BA_ABA_matrices.md [hunk 3]
```diff
@@ -41,12 +41,12 @@ incidence_matrix.axes
 # data: Incidence Matrix
 incidence_matrix.data
 
-# lookup: dictionary linking the branche names and bus numbers with the row 
+# lookup: dictionary linking the branche names and bus numbers with the row
```
Comment (rodrigomha) [inline docs/src/tutorials/tutorial_Incidence_BA_ABA_matrices.md:44]: typo: branche

**Pull request merged**